test/journeys/lib/test-helpers/basic.js

```diff
@@ -0,0 +1,28 @@
+export const elements = {
```

Comment:
do you want the containing folder named space-widget even though these are used across the different widget types (oneOnOne, recents, etc)?